### ISSUE TITLE
508-defect-2 [COLOR]: Links must not rely on color

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#28336]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## Details

Links must not rely on color. Underlines are provided only on hover, which is not available to mobile users.

## Acceptance Criteria

- [ ] Links do not rely on color to convey that they can be interacted with 

## Proposed Solution (if known)

[As discussed in ticket #3556, these heading links should be underlined](https://github.com/department-of-veterans-affairs/va.gov-team/issues/3556).

[Design system ticket #497 has been logged for this](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/497)

## WCAG or Vendor Guidance (optional)

* [WCAG 1.4.1: Use of Color](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)

## Screenshots or Trace Logs

<img width="410" alt="Screen Shot 2021-08-05 at 11 11 24 AM" src="https://user-images.githubusercontent.com/14154792/128374529-e900bce3-9e2c-4e61-8874-122a13db7d31.png">
<img width="411" alt="Screen Shot 2021-08-05 at 11 11 30 AM" src="https://user-images.githubusercontent.com/14154792/128374536-6b6de340-5779-4045-8f60-aa23b189cf10.png">
